target/rivanna/README.md

```diff
@@ -12,7 +12,17 @@ git clone https://github.com/cylondata/cylon.git
 ````
 
 
-## Decveloping with Cylon
+## Developing with Cylon
+
+### Creating a working directory on Rivanna
+
+First, we create a user directory in Rivanna's file system that is backed up so that we do not lose data/programs:
+
+'''bash
+cd /project/$USER
+'''
```

Comment:
for code blocks, you need to use backward-tick char (`) 
```suggestion
```bash
cd /project/$USER
```
```